scripts/aws/cfn/microservice.yml

```diff
@@ -107,6 +107,10 @@ Parameters:
     Type: String
     Description: 'keep track of used projects commitIds'
 
+  NotificationCheckAddress:
+    Type: Boolean
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Parameter NotificationCheckAddress has invalid type Boolean

Reply:
Devi usare
 ```
   Type: String
    Description: ......
    AllowedValues: [True, False]
    Default: 'True'
```